resources/views/entities/pages/posts/_boosted.blade.php

```diff
@@ -0,0 +1,22 @@
+<table class="mb-2">
```

Comment:
I was wondering why this code was so ugly using a table, but then I saw I created this mess in the widgets 😭